src/pages/Index/index.tsx

```
@@ -6,10 +6,44 @@ import logo from "@/shared/assets/logo.svg";
 import { Input } from "@/shared/ui";
 import Box from "@/shared/ui/Box";
 import Spinner from "@/shared/ui/Spinner";
+import { Table } from "@/shared/ui/Table";
 import styles from "@/widgets/Layout/Layout.module.css";
 
 interface Props {}
```

Biome (2.1.2)

[error] An empty interface is equivalent to {}.

Safe fix: Use a type alias instead.


(lint/suspicious/noEmptyInterface)